src/OpenLoco/Vehicles/VehicleManager.cpp

```diff
@@ -124,3 +209,31 @@ namespace OpenLoco::Vehicles::RoutingManager
         return false;
     }
 }
+
+namespace OpenLoco::Vehicles::OrderManager
+{
+    static auto& orders() { return getGameState().orders; }
+    static auto& numOrders() { return getGameState().numOrders; }
+
+    static void sub_470795(const uint32_t removeOrderTableOffset, const int16_t sizeOfRemovedOrderTable)
+    {
+        for (auto head : EntityManager::VehicleList())
+        {
+            if (head->orderTableOffset >= removeOrderTableOffset)
+            {
+                head->orderTableOffset += sizeOfRemovedOrderTable;
+            }
+        }
+    }
+
+    // 0x00470334
+    // Remove vehicle ?orders?
+    void sub_470334(VehicleHead* const head)
```

Comment:
can we rename this method and remove the comment? I think it's clear it's clearing orders?